keyboards/vitamins_included/vitamins_included.h

```diff
@@ -1,10 +1,37 @@
-#ifndef VITAMINS_INCLUDED_H
-#define VITAMINS_INCLUDED_H
+#pragma once
 
 #include "quantum.h"
 
-#include "rev1.h"
+#ifdef KEYBOARD_vitamins_included_rev1
+    #include "rev1.h"
+#else
+    #include "rev2.h"
+#endif
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+	#include <avr/io.h>
+	#include <avr/interrupt.h>
+#endif
+#endif
 
+#define KEYMAP( \
```

Comment:
This macro should actually be named `LAYOUT`.

```suggestion
#define LAYOUT( \
```